client/my-sites/subscribers/components/subscriber-popover/subscriber-popover.tsx

```diff
@@ -30,10 +27,6 @@ const SubscriberPopover = ( {
 	const [ isVisible, setIsVisible ] = useState( false );
 	const onToggle = useCallback( () => setIsVisible( ( visible ) => ! visible ), [] );
 	const buttonRef = useRef< HTMLButtonElement >( null );
-	const site = useSelector( getSelectedSite );
-	const couponsAndGiftsEnabled = useSelector( ( state ) =>
-		getCouponsAndGiftsEnabledForSiteId( state, site?.ID )
```

Comment:
If you do that, you should also rename `getCouponsAndGiftsEnabledForSiteId` to `getCouponsEnabledForSiteId` and all the underlying names

Reply:
I think this can be handled in a follow-up PR. What do you think?

Reply:
I don't see why it would be necessary to have a follow-up PR to rename a function.
I added more feedback in the Slack discussion, as I think another approach might be better,...